modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -2021,6 +2021,584 @@ warn: feature deprecated without replacement.
+[.tabbed-example]
+=====
+[.include-with-neo4j-code]
+======
+Query::
++
+[source,cypher]
+----
+MATCH (n)-[r]->(m)
+WITH r, n.truthCodes AS listOfBooleans
+RETURN listOfBooleans + r:C|D
+----
+
+Description of the returned code::
+`... + r:C|D` is deprecated. It is replaced by `... + (r:C|D)`.
+
+Suggestions for improvement::
+Parenthesize the label expression predicate on the right-hand side of `+`:
++
+[source,cypher]
+----
+MATCH (n)-[r]->(m)
+WITH r, n.truthCodes AS listOfBooleans
+RETURN listOfBooleans + (r:C|D)
+----
+======
+
+[.include-with-GQLSTATUS-deprecated-with-replacement]
+======
+Query::
++
+[source,cypher]
+----
+MATCH (n)-[r]->(m)
+WITH r, n.truthCodes AS listOfBooleans
+RETURN listOfBooleans + r:C|D
+----
+
+Returned GQLSTATUS code::
+01N01
+
+Returned status description::
+warn: feature deprecated with replacement.
+`... + r:C|D` is deprecated.
+It is replaced by `... + (r:C|D)`.
+
+Suggestions for improvement::
+Parenthesize the label expression predicate on the right-hand side of `+`:
++
+[source,cypher]
+----
+MATCH (n)-[r]->(m)
+WITH r, n.truthCodes AS listOfBooleans
+RETURN listOfBooleans + (r:C|D)
+----
+======
+=====
+
+.Using an unescaped variable named `is` as a `WHEN` operand in a simple `CASE` expression
+[.tabbed-example]
+=====
+[.include-with-neo4j-code]
+======
+Query::
++
+[source,cypher]
+----
+MATCH (n)
+WITH n, n.internationalStandard AS is
+RETURN CASE n
+  WHEN is :: INTEGER THEN "ISO/IEC" + is
+  ELSE is
+END AS standardsName
+----
+
+Description of the returned code::
+`WHEN is :: INTEGER` is deprecated. It is replaced by `WHEN ++`is`++ :: INTEGER`.
+
+Suggestions for improvement::
+To continue using variables with this name in simple `CASE` expressions, use backticks to escape the variable name:
++
+[source,cypher]
+----
+MATCH (n)
+WITH n, n.internationalStandard AS `is`
+RETURN CASE n
+  WHEN `is` :: INTEGER THEN "ISO/IEC" + `is`
+  ELSE `is`
+END AS standardsName
+----
+======
+[.include-with-GQLSTATUS-deprecated-with-replacement]
+======
+Query::
++
+[source,cypher]
+----
+MATCH (n)
+WITH n, n.internationalStandard AS is
+RETURN CASE n
+  WHEN is :: INTEGER THEN "ISO/IEC" + is
+  ELSE is
+END AS standardsName
+----
+
+Returned GQLSTATUS code::
+01N01
+
+Returned status description::
+warn: feature deprecated with replacement.
+`WHEN is pass:[::] INTEGER` is deprecated.
+It is replaced by `WHEN ++`is`++ pass:[::] INTEGER`.
+
+Suggestions for improvement::
+To continue using variables with this name in simple `CASE` expressions, use backticks to escape the variable name:
++
+[source,cypher]
+----
+MATCH (n)
+WITH n, n.internationalStandard AS `is`
+RETURN CASE n
+  WHEN `is` :: INTEGER THEN "ISO/IEC" + `is`
+  ELSE `is`
+END AS standardsName
+----
+======
+=====
+
+.Using an unescaped variable named `contains` for relationships in addition or subtraction operations within a `WHEN` operand in a simple `CASE` expression
```

Comment:
```suggestion
.Using an unescaped variable named `contains` in addition or subtraction operations within a `WHEN` operand in a simple `CASE` expression
```

The variable contains in the example bind to an INTEGER (the number of relationships in the path) and that is primarily a characteristics of the specific examples.

As more abstract example would be:
```
WITH 2 AS x, 1 AS contains
RETURN CASE x
  WHEN contains + 1 THEN "foo"
  ELSE "bar"
END AS check
```
This does not involve any nodes and relationships — to make the point that nodes and relationships are not integral to this deprecations.